package_win.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source ./harness-util/package-util.sh
+create_dist_pkg "win" "x86-64" "win_amd64"
```

Comment:
We talked about introducing a fourth parameter, that specifies the launcher for a specific OS, and about introducing an empty (for now) `tic-tac-toe.ps1` for windows. Could you please do these changes in this PR?